packages/vue-language-service/src/plugins/vue-toggle-v-bind-codeaction.ts

```diff
@@ -0,0 +1,78 @@
+import { LanguageServicePlugin } from '@volar/language-service';
+import { VueFile, walkElementNodes } from '@volar/vue-language-core';
+import { NodeTypes } from 'packages/vue-language-core/src/utils/vue2TemplateCompiler';
```

Comment:
`packages/vue-language-core/src/utils/vue2TemplateCompiler` is not a valid path in runtime.

Reply:
Indeed, have no idea why auto-import had to point me to src instead of out